pandas/core/groupby.py

```diff
@@ -3964,13 +4012,117 @@ def filter(self, func, dropna=True, *args, **kwargs):  # noqa
 
 class DataFrameGroupBy(NDFrameGroupBy):
     _apply_whitelist = _dataframe_apply_whitelist
```

Comment:
It is still referenced in `_make_wrapper` and some of the tests.  It might make sense to remove it at the same time as the `__getattr__` change suggested above.